flang/test/Semantics/OpenMP/interop-construct.f90

```diff
@@ -0,0 +1,25 @@
+!RUN: not %flang_fc1 -fopenmp %s 2>&1 | FileCheck %s
+
+subroutine main1()
+  use omp_lib
+  integer(omp_interop_kind) :: obj
+  !CHECK: error: Each interop-var may be speciﬁed for at most one action-clause of each interop construct.
+  !$omp interop init(targetsync,target: obj) use(obj)
+  print *,'pass'
+end
```

Comment:
NIT:
```suggestion
end subroutine test_interop_01
```

Reply:
Addressed the comment on [#1c57d1e](https://github.com/swatheesh-mcw/llvm-project/pull/1/commits/1c57d1ea2b8e7095a834689ad5b832e6a2a84c37)